spec/spec-helpers.js

```diff
@@ -1,8 +1,11 @@
+const os = require('os');
 const http = require('http');
+const https = require('https');
 const proc = require('child_process');
-const StateController = require('../lib/state-controller');
+const OSXSupport = require('../lib/support/osx');
```

Comment:
Right, I should move that part in `spec-helpers.js`. 
I still want the tests to be able to load all the support object. My idea was to allow the tests for all platforms to run on any platform, so that if we make a change in one platform we don't take the risk to ship a broken version for the other platforms because their tests didn't run in the CI. But I can make the support object evaluated lazily and cached and then in the tests I can just clear that cache when testing for a different platform.